types/inboxsdk/index.d.ts

```diff
@@ -754,43 +757,45 @@ declare namespace InboxSDK {
       onClick?: () => void;
     }
 
-    type NativeRouteIDs =
-      'INBOX' |
-      'ALL_MAIL' |
-      'SENT' |
-      'STARRED' |
-      'DRAFTS' |
-      'SNOOZED' |
-      'DONE' |
-      'REMINDERS' |
-      'LABEL' |
-      'TRASH' |
-      'SPAM' |
-      'IMPORTANT' |
-      'SEARCH' |
-      'THREAD' |
-      'CHATS' |
-      'CHAT' |
-      'CONTACTS' |
-      'CONTACT' |
-      'SETTINGS' |
-      'ANY_LIST';
-
-    type NativeListRouteIDs =
-      'INBOX'
-      | 'ALL_MAIL'
-      | 'SENT'
-      | 'STARRED'
-      | 'DRAFTS'
-      | 'SNOOZED'
-      | 'DONE'
-      | 'REMINDERS'
-      | 'LABEL'
-      | 'TRASH'
-      | 'SPAM'
-      | 'IMPORTANT'
-      | 'SEARCH'
-      | 'ANY_LIST';
+    enum NativeRouteIDs {
```

Comment:
Just as a heads up, this means that anyone that was previously passing in the strings will be broken. Maybe a better idea would be to extract the non-enum-versions of the values to preserve compatibility

```ts
keyof Record<keyof typeof NativeRouteIDs, any>
```

Reply:
I'm not sure that anyone had some code with those strings because it doesn't work as you can see in the forum post that I added above.

Reply:
Ah, I see. I can merge this in then.